### ISSUE TITLE
Internal Constructor from another project appears in autocomplete/intellisense.

### ISSUE THREAD

**Issue by bryanwood** [OmniSharp/omnisharp-vscode#1664]
## Environment data
`dotnet --info` output:
```
.NET Command Line Tools (2.0.0-preview2-006497)

Product Information:
 Version:            2.0.0-preview2-006497
 Commit SHA-1 hash:  06a2093335

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.15063
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.0.0-preview2-006497\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0-preview2-25407-01
  Build    : 40c565230930ead58a50719c0ec799df77bddee9
```
VS Code version: `1.14.2`
C# Extension version: `1.11.0`

## Steps to reproduce
Have two projects, one with:
```
using System;

namespace Lib
{
    public class Class1
    {
        public Class1() { }

        internal Class1(string param) { }
    }
}

```
and another with 
```
using System;

namespace App
{
    class Program
    {
        static void Main(string[] args)
        {
            var lib = new Lib.Class1();
            var lib2 = new Lib.Class1("param");
        }
    }
}

```

## Expected  behavior

Internal ctor is not visible in autocomplete/intellisense.

## Actual behavior

Internal ctor is visible in autocomplete intellisense.

It does squiggle the internal ctor as an error if used.

Example here:
[https://github.com/bryanwood/omnisharp-vscode-internal-ctor-repro](https://github.com/bryanwood/omnisharp-vscode-internal-ctor-repro)

**Comment by DustinCampbell** [comment]
Thanks for reporting this. Much appreciated!

**Comment by rchande** [comment]
@akshita31 Another one for you to take a look at.

**Comment by akshita31** [comment]
This issue has been fixed with the change #https://github.com/OmniSharp/omnisharp-roslyn/pull/1030 .

**Comment by akshita31** [comment]
This issue has been fixed in the 1.14 milestone. We just released a beta of 1.14 - https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.14.0-beta4. Follow [these instructions](https://github.com/OmniSharp/omnisharp-vscode/wiki/Installing-Beta-Releases) to install a beta release of C# for VS Code.
